### PR TITLE
API 2200 support for Network resources

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
- oneview_connection_template
- oneview_ethernet_network
- oneview_fc_network
- oneview_fcoe_network
- oneview_network_set
- oneview_scope
- oneview_uplink_set

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+### Notes
+- This release extends support of the SDK to OneView REST API version 2200 (OneView v5.50).
+
+### Oneview Features supported
+- oneview_connection_template
+- oneview_ethernet_network
+- oneview_fc_network
+- oneview_fcoe_network
+- oneview_network_set
+- oneview_scope
```
Comment (VenkateshRavula) [inline CHANGELOG.md:11]: Since you added library files, add storage system if it is validated..

File: endpoints-support.md [hunk 2]
```diff
@@ -139,23 +139,23 @@
...
+|<sub>/rest/network-sets/{id}</sub>                                                       | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets/{id}</sub>                                                       | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/network-sets/{id}</sub>                                                       | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:  | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: Added twice..

File: CHANGELOG.md (hunk 1 above)
Comment (yuvirani) [reply CHANGELOG.md:11]: Yes, validation done. I will add as part of storage PR

**Review by VenkateshRavula — APPROVED** [review]

File: endpoints-support.md (hunk 2 above)
Comment (yuvirani) [reply endpoints-support.md]: Done

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**